frontend/components/Pagination/Pagination.tsx

```diff
@@ -4,41 +4,46 @@ import classnames from "classnames";
 import Button from "components/buttons/Button";
 import Icon from "components/Icon";
 
-const baseClass = "pagination-new";
+const baseClass = "pagination";
 
-interface IPaginationProps {
+export interface IPaginationProps {
+  /**  Disable next page is usually passed through from api metadata, or on loading */
   disableNext?: boolean;
+  /**  Disable prev page is usually passed through from api meta data, on page 0, or on loading */
   disablePrev?: boolean;
-  onNextPage?: () => void;
-  onPrevPage?: () => void;
+  onNextPage: () => void;
+  onPrevPage: () => void;
   className?: string;
+  /** UI Pattern: Hide pagination iff there's one page of results */
```

Comment:
Update comment to remove if and only if, since there's other reasons pagination shouldn't be rendered